src/styles.css

```diff
@@ -0,0 +1,5092 @@
+  color: var(--gray-800);
+  border: solid 1px currentColor;
+}
+.btn-outline:hover {
+  /* color: var(--gray-50); */
+  background-color: var(--gray-200);
+  /* box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px; */
+}
+
+.btn-text {
+  margin-left: 0.375em;
+}
+
+/* button rounded */
+.btn-rounded {
+  border-radius: 9999px;
+}
+
+/* button floating */
+.btn-float {
+  width: 4rem;
+  height: 4rem;
+  border-radius: 50%;
+  border: none;
+  color: var(--gray-50);
+  background-color: var(--gray-800);
+  transition: 0.3s;
+}
+.btn-float:hover {
+  background-color: var(--gray-700);
+  /* box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px; */
+}
+
+.btn-float-outline {
+  background: inherit;
+  color: var(--gray-800);
+  border: solid 1px currentColor;
+}
+.btn-float-outline:hover {
+  color: var(--gray-50);
+  background-color: var(--gray-700);
+  /* box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px; */
+}
+
+/* predefined colorful buttons */
+/* Red */
+/* Green */
+/* yellow */
+/* Blue */
+
+/* red */
+.btn-red {
+  background-color: var(--red-500);
+}
+.btn-red:hover {
+  background-color: var(--red-600);
+}
+
+/* green */
+.btn-green {
+  background-color: var(--green-500);
+}
+.btn-green:hover {
+  background-color: var(--green-600);
+}
+
+/* yellow */
+.btn-yellow {
+  background-color: var(--yellow-500);
+}
+.btn-yellow:hover {
+  background-color: var(--yellow-600);
+}
+
+/* blue */
+.btn-blue {
+  background-color: var(--blue-500);
+}
+.btn-blue:hover {
+  background-color: var(--blue-600);
+}
+
+/* cards */
+/* e-commerce card */
+.card {
+  width: 250px;
+  transition: 0.3s ease;
+}
+.card:hover {
+  box-shadow: rgba(149, 157, 165, 0.2) 0px 8px 24px;
+}
+
+.card-text {
+  min-width: 300px;
+  max-width: 500px;
+  border-radius: 5px;
+}
+.card-text img {
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
+}
+.model-name {
+  margin-bottom: 0.5rem;
+  color: var(--gray-500);
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  overflow: hidden;
+  width: 100%;
+}
+
+.brand-name,
+.card-text-title {
+  font-weight: 600;
+  font-size: 1.125rem;
+  line-height: 1.75rem;
+  margin-bottom: 0.5rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card-text-title {
+  font-size: 1.25rem;
+  line-height: 1.75rem;
+}
+
+.w-250 {
+  width: 250px;
+}
+
+.card-text-body {
+  margin-bottom: 0.5rem;
+  color: var(--gray-600);
+  font-size: 1rem;
+  line-height: 1.5rem;
+}
+
+.wishlist-icon {
+  font-size: 1.5rem;
+  line-height: 2rem;
+}
+.wishlist-icon:hover {
+  cursor: pointer;
+}
+.price {
+  font-weight: 600;
+  margin-bottom: 0.5rem;
+}
+.mrp {
+  color: var(--gray-500);
+  font-size: 0.875rem;
+  line-height: 1.25rem;
+  font-weight: 400;
+  margin-right: 0.5rem;
+  text-decoration: line-through;
+}
+.offer {
+  color: var(--red-400);
+  font-size: 0.875rem;
+  line-height: 1.25rem;
+  font-weight: 400;
+  margin-right: 0.5rem;
+}
+
+.prod-image {
+  position: relative;
+}
+.prod-badge {
```

Comment:
U can rename like "trending-tag" !! Better readability while going through the code !! No need to look for like what class is been given to trending ! The trending-tag itself makes sense that it belongs to trending it will click in mind fast!!
Or you can rename like "card-trending-tag" it will provide more understanding!!
Again upto you !! Just putting down my views of how one can understand the code in a go !!